src/components/ChangeTag/ChangeTag.tsx

```diff
@@ -0,0 +1,33 @@
+import { ArrowDownIcon, ArrowUpIcon } from '@chakra-ui/icons'
+import { Skeleton } from '@chakra-ui/react'
+import type { TagProps } from '@chakra-ui/tag'
+import { Tag } from '@chakra-ui/tag'
+import { bnOrZero } from '@shapeshiftoss/chain-adapters'
+import { Amount } from 'components/Amount/Amount'
+
+export type ChangeTagProps = {
+  value?: string | number
+  hasBackground?: boolean
+} & TagProps
+
+export const ChangeTag: React.FC<ChangeTagProps> = ({ value, hasBackground = false, ...props }) => {
```

Comment:
naming suggestion: restProps, rest, or tagProps

Reply:
Makes sense ser! Fixed it using `tagProps`